.github/workflows/pr-ci-ok-only.yml

```
@@ -0,0 +1,39 @@
+# This workflow is a version of `pr.yml` that runs in exactly the cases that `pr.yml` doesn't. That is, where `pr.yml`
+# skips execution due to `paths-ignore`, this workflow runs only in those cases. We do this so that we can always
+# produce a `ci-ok` check, which we configure as a required passing step on all PRs to be merged into the repository.
+#
+# This is an officially blessed pattern for handling skipped but required status checks. See
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+# for more information.
+
+on:
+  pull_request:
+    paths-ignore:
+      # Skip all files *except CHANGELOG* changes.
+      - '**'
+      - '!CHANGELOG.md'
+      - '!CHANGELOG_PENDING.md'
+  push:
+    branches:
+      - main
+    tags:
+      - sdk/v*.*.*
+    paths-ignore:
+      # Skip all files *except CHANGELOG* changes.
+      - '!CHANGELOG.md'
+      - '!CHANGELOG_PENDING.md'
```

```suggestion
      - '**'
      - '!CHANGELOG.md'
      - '!CHANGELOG_PENDING.md'
```

If we want this to run for pushes to `main` we first need to include `**` before excluding the other files.